examples/monitors/metric-alert/payload.json

```diff
@@ -0,0 +1,28 @@
+{
+    "tag": "prod-us-west",
+    "agg_func": "sum",
+    "agg_window": "1h",
+    "lookback": "12h",
+    "rule": "api_access",
+    "stat_types": [
+        "count"
+    ],
+    "measurement": "api_access.count",
+    "source_filters": {
+        "sourcetype": "K8s"
```

Comment:
you can add more:
"source_filters": {
    "sourcetype": "K8s",
    "k8s_namespace": "default",
    "k8s_container_name": "admin",
    "k8s_controller_kind": "ReplicaSet",
    "k8s_controller_logical_name": "admin-api-deployment"
  },

Reply:
added the values: cc2d024